extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -281,6 +282,25 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
     );
   }
 
+  public async cancelVariantAnalysis(variantAnalysisId: number) {
+    const variantAnalysis = this.variantAnalyses.get(variantAnalysisId);
+    if (!variantAnalysis) {
+      throw new Error(`No variant analysis with id: ${variantAnalysisId}`);
+    }
+
+    if (!variantAnalysis.actionsWorkflowRunId) {
+      throw new Error(`No workflow run id for variant analysis ${variantAnalysis.query.name}`);
```

Comment:
```suggestion
      throw new Error(`No workflow run id for variant analysis ${variantAnalysis.id}`);
```
Is it better to print the ID or the query name? I think in other places we use the ID to identify the variant analysis, even in error messages that may be visible to the user. This error hopefully shouldn't ever trigger unless there's a bug in the code, because we shouldn't show the option to cancel it if there's no actions workflow run ID yet.